helix-core/src/test/java/org/apache/helix/LogHelper.java

```diff
@@ -0,0 +1,26 @@
+package org.apache.helix;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.apache.logging.log4j.core.config.Configuration;
+import org.apache.logging.log4j.core.config.LoggerConfig;
+
+/**
+ * Helper class for Logging related stuff.
+ */
+public class LogHelper {
```

Comment:
This is not needed for new class creation. Just move the function as a private function in that test class.

Reply:
Hmmm I made a new class, just so that in future if anybody needs to enable the logs for the test, they can simply use this class rather than starting from scratch. So, should I make it private or keep the helper class?